pages/agent/v3/elastic_ci_aws.md

```diff
@@ -194,10 +194,10 @@ CloudWatch Logs and EC2 instance log data are forwarded to CloudWatch Logs, but
 <!-- vale off -->
 
 * [Amazon Linux 2](https://aws.amazon.com/amazon-linux-2/)
-* [Buildkite Agent v3.43.1](https://buildkite.com/docs/agent)
-* [Git v2.39.1](https://git-scm.com/)
-* [Docker](https://www.docker.com) - 20.10.7 (Linux) and 20.10.0 (Windows)
-* [Docker Compose](https://docs.docker.com/compose/) - 1.29.2 (Linux) and 1.28.5 (Windows)
+* [Buildkite Agent v3.44.0](https://buildkite.com/docs/agent)
+* [Git](https://git-scm.com/)
```

Comment:
What's the reason for dropping the Git version mentioned?

Reply:
I meant to check if the version changed, and get back to this.
We switched back to not manually installing it.
While I was putting the version back, I added Git LFS which people might be interested in.